include/deal.II/grid/grid_generator.h

```diff
@@ -60,6 +61,183 @@ namespace GridGenerator
    */
   ///@{
 
+  /**
+   * Namespace Airfoil contains classes and functions in order to create a
+   * C-type mesh for the (flow) field around Joukowski or NACA airfoils.
+   *
+   * @ingroup GridGenerator
+   */
+  namespace Airfoil
```

Comment:
I realize one thing now: Can you move this code towards the end of the file? I think it makes sense to keep the basic grids like the hypercube at the top and the more specialized ones further down.